src/Randomizer.SMZ3.Tracking/VoiceCommands/SpoilerModule.cs

```diff
@@ -483,11 +486,12 @@ private bool GiveLocationSpoiler(Location location)
 
         private bool GiveItemLocationSpoiler(ItemData item)
         {
-            var progression = Tracker.GetProgression();
+            var zeldaProgression = Tracker.GetProgression(assumeKeys: !_config.ZeldaKeysanity);
+            var metroidProgression = Tracker.GetProgression(assumeKeys: !_config.MetroidKeysanity);
```

Comment:
I don't really like this solution. Since we're doing this all over the place now, maybe we should add Region as parameter to GetProgression

Reply:
Fixed. I was just able to use the new GetProgression that I already had with the IHasLocations parameter.

Progression may be a good thing to pull out into its own service in the future tbh.

Reply:
Considering Progression is just your tracked items, shouldn't it be part of the ItemService?

Reply:
Ah, yeah. That would probably make more sense.